app/assets/css/main.styl

```diff
@@ -15,6 +15,8 @@ body
 @media only screen and (max-device-width: 500px)
```

Comment:
I think we should *only* show the banner on screen media with a somewhat wider screen width than this (maybe 800px). Basically anytime the right navigation sidebar doesn't appear on the right it obscures the documentation.

(on another note we should probably make the navigation fall *below* the content when the screen is too small to fit it on the right)